src/App.css

```diff
@@ -1,42 +1,38 @@
-#root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
+.App {
   text-align: center;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
+.App-logo {
```

Comment:
We can do some more cleanup in the app style, especially related to the logo.

Reply:
Okay, yeah. I can remove the styling for the logo and re-order the `.App` and `.App-header`styling!